### PR TITLE
Update activation.md

### PR DESCRIPTION
**Pull request by Vivraan**
#### Changes

- Change syntax (installment -> installation)

#### Checklist

- [x] Read the contribution [guide](../CONTRIBUTING.md) and accept the [code](../CODE_OF_CONDUCT.md) of conduct
- [ ] Readme (updated or not needed)
- [ ] Tests (added, updated or not needed)

### CODE REVIEW

**Comment by webbertakken** [comment]
This is no longer correct. It uses an installment. It doesn't need the user to install anything. We should not make it sound like the user is providing the installation of unity.

The second part is fine.

**Comment by Vivraan** [comment]
Could you explain the difference in terminology? I'm assuming instalment (US: installment) is a container-specific jargon that I'm not aware of.

**Comment by webbertakken** [comment]
I would say both are correct. I was trying to get at the part where it now sounds like you have to provide the unity version yourself, especially because of the "need a" part. 

The point is actually that we don't want the user to worry in any way about what the action needs, by only providing parameters.

File: docs/github/activation.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Activation
 
-All actions utilize an installment of Unity, and as such need to be activated.
+All actions need a Unity installation, which needs to be activated.
```
Comment (webbertakken) [inline docs/github/activation.md:3]:
```suggestion
All actions use a Unity installation, which needs to be activated.
```

**Comment by webbertakken** [comment]
I found out in [this video](https://youtu.be/vSdSFxIKy5w?t=266) that it's possible to commit suggestions directly from the PR, so I made the tiny change myself.

Thanks a lot for your contribution.

**Pull request merged**